### PR TITLE
chore: update CHANGELOG and README

### PR DESCRIPTION
**Pull request by sanderblue**
Release notes for `1.6.0`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Requirements
 ------------
 
 -	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
--	[Go](https://golang.org/doc/install) 1.8 (to build the provider plugin)
+-	[Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
```
Comment (ctrombley) [inline README.md]: I wouldn't mind getting rid of this line and keeping everything above the fold targeted toward users (not developers). We specify the 1.13 requirement down below.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Requirements
...
 
 Building The Provider
```
Comment (ctrombley) [inline README.md]: Likewise here - can we move this whole section down below "Using the provider"?  Maybe move it into the build section below?

File: README.md [hunk 3]
```diff
@@ -50,27 +53,28 @@ $ $GOPATH/bin/terraform-provider-newrelic
...
+In order to test the provider, run `make test`. This will run the unit test suite.
 
 ```sh
 $ make test
 ```
 
 In order to run the full suite of Acceptance tests, run `make testacc`.
 
-*Note:* Acceptance tests create real resources, and often cost money to run.
+*Note:* Acceptance tests *create real resources*, and often cost money to run. The environment variable `NEWRELIC_API_KEY` must also be set with your key for acceptance tests to work properly.
```
Comment (ctrombley) [inline README.md]: `NEWRELIC_LICENSE_KEY` is also currently needed for acc testing.

File: README.md (hunk 1 above)
Comment (sanderblue) [reply README.md]: Yeah, I can understand this change. I kind of pondered that a bit earlier as well.

File: README.md (hunk 3 above)
Comment (sanderblue) [reply README.md]: Ah, right. Thanks.

**Pull request merged**